packages/client/src/typescript.ts

```diff
@@ -109,6 +110,16 @@ export interface ITsConstructor extends ITsDocBase, ITsCallable {
     kind: Kind.Constructor;
 }
 
+export interface ITsAccessor extends ITsDocBase {
+    kind: Kind.Accessor;
+    /** If a set signature is defined and documented for this accessor, this will contain its documentation. */
+    getDocumentation: IBlock | undefined;
+    /** If a get signature is defined and documented for this accessor, this will contain its documentation. */
+    setDocumentation: IBlock | undefined;
+    /** Type of the accessor, or null if neither a get or a set accessor is defined. */
+    type: string | null;
```

Comment:
when would this actually be `null`? why would we ever generate a `ITsAccessor` object in that case?

Reply:
You're right, sorry that was from my code before where I had seperate declarations for get and set accessors. Changed in the updated PR.